documentation/running_bolt_commands.md

```diff
@@ -1,418 +1,640 @@
-    ssh://root:password@hostname, pcp://host01, winrm://Administrator:password@hostname
-    ```
+### Specifying connection credentials
 
+To establish connections with remote targets, Bolt will need to provide
+credentials to the target. These credentials can be provided at the command
+line or in an inventory file, and the credentials you provide may vary based
+on the operating system the target is running.
 
-#### Specify targets from an inventory file
+Whether a target is running a Unix-like operating system or Windows, the
+simplest way to specify credentials is to pass the `user` and `password`
+to the Bolt command:
 
-To specify targets from an inventory file, reference targets by target name, a
-glob matching names in the file, or the name of a group of targets.
--   To match all targets in both groups listed in the inventory file example:
-    ```
-    --targets elastic_search,web_app
-    ```
--   To match all the targets that start with "elasticsearch" in the inventory
-    file example:
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'pwd' --targets servers --user bolt --password puppet
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'pwd' -Targets servers -User bolt -Password puppet
+  ```
+
+If you would prefer to have Bolt securely prompt for a password, so that it
+does not appear in a process listing or on the console, use the
+`password-prompt` option instead:
+
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'pwd' --targets servers --user bolt --password-prompt
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'pwd' -Targets servers -User bolt -PasswordPrompt
+  ```
+
+### Specifying a transport
+
+When Bolt establishes a connection with a target, it uses a specific transport
+to do so. By default, Bolt will connect to a target using the `ssh` transport.
+However, you can specify a specific transport that Bolt should use to connect
+to each target.
+
+You can specify the transport used to connect to a specific target by setting
+it as the protocol in the target's URI:
+
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'Get-Location' --targets winrm://windows.example.org
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'Get-Location' -Targets winrm://windows.example.org
+  ```
+
+You can also change the default transport used to connect to targets that do
+not specify a transport in their URI or in their inventory configuration
+using the `transport` command-line option:
+
+- _\*nix shell command_
+
+  ```shell
+  bolt command run 'Get-Location' --targets windows.example.org --transport winrm
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltCommand -Command 'Get-Location' -Targets windows.example.org -Transport winrm
+  ```
+
+📖 **Related information**
+
+- [Bolt transports reference](bolt_transports_reference.md)
+
+## Interacting with remote targets
```

Comment:
The first section here on running a command has already been covered further up in the doc, so I don't think it's helpful to repeat it here. Maybe we should pull the "Specifying" sections above out of the syntax section - just make them H2s and their children H3s. That would make it a bit easier for people to navigate to those using the mini-TOC. And then this section can just start with scripts.

Reply:
Decided to remove the `command run` example at the beginning of the page. It seems cleaner to have all of the action commands grouped together, especially since `command run` has additional information describing how to run commands from files and stdin.